supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -594,36 +594,14 @@ Use "Red Hat Virtualization Manager". Spell out in full for the first instance i
 [discrete]
 [[region]]
 ==== image:images/yes.png[yes] region (noun)
-*Description*: In Red Hat Ceph Storage, a region is the deprecated term for referring to a zone group. Red Hat Ceph Storage 1.3 uses regions.
+*Description*: In Red Hat Ceph Storage, a _region_ is the deprecated term for referring to a zone group. Red Hat Ceph Storage 1.3 uses regions.
 
 *Use it*: yes
 
 *Incorrect forms*:
 
 *See also*: xref:zone-group[zone group]
 
```

Comment:
Didn't we just add "regex" and "regular expression" to the glossary? Not sure why they are being deleted in this PR.

Reply:
Re-added these files. I need to take more care with resolving merge conflicts with VSC. 

@bburt-rh . You OK to approve the PR?

Reply:
@dfitzmau With these fixes in, I'll approve it now.